peer/brontide.go

```diff
@@ -702,6 +702,30 @@ func (p *Brontide) loadActiveChannels(chans []*channeldb.OpenChannel) (
 			}
 
 			msgs = append(msgs, chanSync)
+
```

Comment:
ok so just to double check: we are meant to be sending both ChanSync _and_ Shutdown?  like if peer gets shutdown and then we restart and first resend ChanSync before shutdown, is that fine? I think it is... just want to double check

Reply:
cool, thanks 👍